looprpc/client.proto

```diff
@@ -822,6 +822,14 @@ message LiquidityParameters {
     */
     uint64 autoloop_budget_sat = 11;
 
+    /*
+    The start time for autoloop budget, expressed as a unix timestamp in
+    seconds. If this value is 0, the budget will be applied for all
```

Comment:
Doesn't `deprecated` serve that role?

I can also add "DEPRECATED" on the comment too

Reply:
It would be good to add "Deprecated, use autoloop_budget_start_sec." in the comment as this will also be surfaced in the API docs website. I've seen it done this way for many other RPC fields such as [EstimateFeeResponse](https://github.com/lightningnetwork/lnd/blob/d44823f6e8580d2fa5f193a5382be351610f7c2b/lnrpc/lightning.proto#L1098).

Reply:
Added the deprecated note in the proto comment